orttraining/orttraining/test/training_ops/cuda/cross_entropy_test.cc

```diff
@@ -313,7 +313,7 @@ static void TestSoftmaxCrossEntropyLoss(const std::vector<int64_t>* X_dims,
 TEST(CudaKernelTest, SoftmaxCrossEntropyLoss_TinySizeTensor) {
   std::vector<int64_t> X_dims{8, 2};
   std::vector<int64_t> index_dims{8};
-  std::vector<int64_t> weight_dims{8};
+  std::vector<int64_t> weight_dims{2};
```

Comment:
changed by accident?

Reply:
No, weights, should be 2, it should be same shape as C